### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by trapacska**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
...
+  }]
+}
+```
+
+## Basic step logic
+
+Based on the model and available infos above.
+
+- Create a step which has one input for Bitrise personal access token and one output (PREVIOUS_BUILD_FAILED for example)
+- The step should find out what trigger_event_type if the actual build:
```
Comment (lpusok) [inline README.md:52]: What if build started manually?

File: README.md [hunk 2]
```diff
@@ -1 +1,57 @@
...
+Based on the model and available infos above.
+
+- Create a step which has one input for Bitrise personal access token and one output (PREVIOUS_BUILD_FAILED for example)
+- The step should find out what trigger_event_type if the actual build: 
+  - BITRISE_GIT_TAG set = tag
+  - BITRISE_PULL_REQUEST set = pull request
+  - None of the above = push
+- The step should request builds with query parameters(at least): branch, trigger_event_type.
+- Find the previous build matching the current build parameters(if PR then PR ID for example, etc...) and check it's status.
+- If the previous build is failed then export an env, for example: `PREVIOUS_BUILD_FAILED=true`
```
Comment (lpusok) [inline README.md]: If previous build aborted, how will handle that?

File: README.md [hunk 3]
```diff
@@ -1 +1,57 @@
-# temp-01
\ No newline at end of file
+## Infos (relevant envs) available in the currently running build (in which the step runs)
+
+- BITRISE_APP_SLUG
+- BITRISE_BUILD_NUMBER
+- BITRISE_GIT_BRANCH
+- BITRISEIO_GIT_BRANCH_DEST
+- BITRISE_GIT_TAG
+- BITRISE_PULL_REQUEST
```
Comment (lpusok) [inline README.md:8]: Could add a new env var, which is set by website.

**Review by BirmacherAkos — COMMENTED** [review]
I have the same comments like Laci

File: README.md (hunk 2 above)
Comment (godrei) [reply README.md]: what about `PREVIOUS_BUILD_STATUS`
- <not set/empty value> - no prev build for the same start event
- success
- failed
- aborted

**Review by godrei — CHANGES_REQUESTED** [review]
Also please add example, how the actual build's steps would use the exposed env

**Review by lszucs — COMMENTED** [review]
Nothing to add to previous comments. Looks good generally though!

File: README.md (hunk 3 above)
Comment (trapacska) [reply README.md:8]: ?

**Review by lszucs — COMMENTED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -51,7 +52,16 @@ Based on the model and available infos above.
 - The step should find out what trigger_event_type if the actual build: 
   - BITRISE_GIT_TAG set = tag
   - BITRISE_PULL_REQUEST set = pull request
-  - None of the above = push
+  - BITRISE_GIT_COMMIT + none of the above = push
+  - everything else - manual
 - The step should request builds with query parameters(at least): branch, trigger_event_type.
 - Find the previous build matching the current build parameters(if PR then PR ID for example, etc...) and check it's status.
-- If the previous build is failed then export an env, for example: `PREVIOUS_BUILD_FAILED=true`
+- If the previous build is failed then export an env witht he status, for example: `PREVIOUS_BUILD_STATUS=[success or failed or aborted]`
```
Comment (BirmacherAkos) [inline README.md]: typo: export an env **witht he** status => export an env **with the** status

**Review by BirmacherAkos — CHANGES_REQUESTED** [review]
typo

**Review by BirmacherAkos — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (lpusok) [reply README.md:8]: I mean implement change detection on website logic.

File: README.md (hunk 1 above)
Comment (lpusok) [reply README.md:52]: Isn't BITRISE_GIT_COMMIT always set? Meaning is there a difference between push and manual?

**Review by lpusok — COMMENTED** [review]
left comment

File: README.md (hunk 1 above)
Comment (trapacska) [reply README.md:52]: not always set. if you start a build you start it by branch

**Review by godrei — APPROVED** [review]

**Review by BirmacherAkos — APPROVED** [review]

**Review by lpusok — COMMENTED** [review]
ok

**Review by lszucs — COMMENTED** [review]
LGTM

**Pull request merged**